code/modules/admin/verbs/communications_panel.dm

```diff
@@ -0,0 +1,71 @@
+GLOBAL_LIST_EMPTY(announcements)
+GLOBAL_LIST_EMPTY(messages)
```

Comment:
Global lists all share the same namespace, right? Maybe a more detailed name than just 'messages' would be better here? Since I could imagine someone else wanting a different global list with the name messages.